docs/user-guide/cases.md

```diff
@@ -22,13 +22,29 @@ It can be visualized by clicking on the 'Visualize report' link on the left side
 ### Pinned variants
 There's a list of "Strong candidates" on the page. This list is curated by the collaborators and consists of "pinned" variants from the variants list. This feature can be used to mark variants of particular interest that you might want to highlight to other users. Pinned variants for cases with at least one assigned HPO phenotype can be used to produce Clinvar submission files (See more under "Variant").
 
+### Matching causatives
+The case page shows a list of matching causative variants from other cases in a foldable tab. By default only the number of matches is displayed. The user can click to open the tab and expose links to the variant in the case where it has been marked causative.
+
+### Managed variants
+The case page shows a list of matching managed variants from other cases in a foldable tab. By default only the number of matches is displayed. The user can click to open the tab and expose a list of managed variants found in the case.
+
 ### Misc.
-The case page also displays a simple wiki-style synopsis on the current case that can be used to communicate information on the case to collaborators.
+The case page also displays a simple wiki-style synopsis on the current case that can be used to communicate information on the case to collaborators. The synopsis and other fields on the case page can be populated directly on load - see the admin section.
 
 Rank model version shows which rank model that has been used to rank the variants in the case
 
 The "Activity" feed is a reverse chronological list of events and comments related to the case. Examples of events include comments, status updates, assignments, Sanger sequencing orders etc.
 
+### Cytogenomics
+The case page contains three (admin configuration optional) cytogenomics applications: [Gens](https://github.com/Clinical-Genomics-Lund/gens), [Chromograph](https://github.com/mikaell/chromograph) and [vcf2cytosure](https://github.com/NBISweden/vcf2cytosure).
+Gens integration is available via the "View CN profile" button on the case page as well as the "Copy number profile" links under SV variant summary on each SV variant page.
+Chromograph integration can be found under the Images cards on the case page. Chromograph will display ideograms, genome coverage from [TIDDIT](https://github.com/SciLifeLab/TIDDIT) (orange graph), autozygosity from [rhocall][rhocall] (gray bars) and UPD calls from [UPD.py](https://github.com/bjhall/upd). UPD calls are colored deep red for maternal UPD candidate regions,
```

Comment:
You could add these links to then end of the page, like it is done for rhocall if you want to.

Reply:
Yeah, I use the logic of direct link for one mention, reference for two. I'm more worried about the images - seems the current .io doesn't show the others from the same page?

Reply:
Also images from other pages like [dashboard](https://clinical-genomics.github.io/scout/user-guide/dashboard/) are missing.

Reply:
Checking the documentation looks like we should add each image of the page to the yaml index (mkdocs.yml), which we don't:
https://www.mkdocs.org/user-guide/writing-your-docs/#linking-to-images-and-media

Reply:
Thanks, that link would have saved me some googling had I seen it! 😅 Anyway, still needed to come to terms with the abs/relative URL issue between local testing and deployment. For local testing, all the `[](/img/sth.png)` worked fine, but they fail when the site url for the same is `io.github.com/scout/img/sth.png`.

Reply:
Nice with the doc pages fixing!